_posts/0000-01-02-darth-poo.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
github learning not working across browser tabs!

Reply:
i-cant-get-no, sa-tis-faction ta ta, ta-ta-ta, ta-ta ta-ta, ta ta

Reply:
i keep losing the instructions from Github Learning. This is retarded. Circular waste of time.